docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -127,17 +150,72 @@ PAF-of-1 Cause/Risk Pairs
 Risk-Outcome Relationships
 ++++++++++++++++++++++++++
 
-Coverage Gap Framework
-++++++++++++++++++++++
+Stratifying Exposure Variables by Baseline Intervention Coverage
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+From GBD we obtain overall population estimates for our outcomes of interest
+(prevalence of vitamin A deficiency, birth prevalence of neural tube defects,
+distribution of haemoglobin levels, and distribution of birth weights). Because
+there exists baseline coverage of our interventions --- i.e. there is already
+some food fortification occurring in the locations we are simulating --- these
+overall population estimates include both people who are eating fortified food
+and those who are not.
+
+When we introduce additional food fortification into the population, only those
+who are *not* already eating fortified food should receive the benefit.
+Therefore, in order to properly measure the effects of our interventions, we
+must first stratify our population into those who are already covered vs. not
+covered by the forticant of interest. We then need to calculate the risk
+exposure (or other relevant outcome) in each coverage stratum to get a complete
+picture of the baseline population. For example, we will need to know the
+prevalence of vitamin A deficiency or the distribution of haemoglobin in the
+subpopulation already eating fortified food as well as in the subpopulation not
+eating fortified food.
+
+These coverage-stratified exposures are calculated so that:
+
+1.  Their population-weighted average matches the overall population estimate
```

Comment:
```suggestion
1.  Their population-weighted average of the risk exposure matches the overall population estimate
```

Reply:
I started editing things to just say "exposure" instead of "risk exposure" because it's not always risks we're talking about, but possibly other outcomes like neural tube defects.

But I can reword this sentence to explicitly say "exposures" instead of implicitly referring to them with "Their".